addons/web_editor/static/src/components/media_dialog/document_selector.js

```diff
@@ -64,11 +64,16 @@ export class DocumentSelector extends FileSelector {
             linkEl.href = href;
             linkEl.title = attachment.name;
             linkEl.dataset.mimetype = attachment.mimetype;
+            linkEl.dataset.fileName = attachment.name;
+            linkEl.textContent = attachment.name;
+            // The document behaviour is by default set to open in a new tab.
+            // This behaviour is configurable with an option in the Link Tools.
+            linkEl.target = '_blank';
             return linkEl;
         }));
     }
 }
-DocumentSelector.mediaSpecificClasses = ['o_image'];
```

Comment:
Is the `o_image` class still used anywhere else? If not, I guess we can keep it for compatibility but it should be explained next to the related SCSS rules.

Reply:
Yes, it is still used, at least for knowledge files

Reply:
Indeed, then ok :+1:

Reply:
@younn-o If you edit an old o_image (which actually is still used as you explained), it's problematic:
- if you change to another document, you keep the icon but text is added
- if you change to an image, the image dimensions are forced to the o_image one

Can't we improve that? I am not sure I am confident merging this change in 16.0, the first thing I tried is not right :/ Also, is not there a way to avoid the enterprise PR and the change in knowledge?

Reply:
The o_image class is now removed by default to tackle documents uploaded before this fix.
Unfortunately, with this change in the MediaDialog api, the knowledge override must be adapted...
I rebased https://github.com/odoo/odoo/pull/76313 - which targets master - on top of this fix. If the change in the Media Dialog API is too risky for stable, this fix can always be included in the mentioned PR